source/client-side-operations-timeout/etc/templates/deprecated-options.yml.template

```diff
@@ -0,0 +1,233 @@
+description: "operations error if timeoutMS is used with a deprecated timeout option"
+
+schemaVersion: "1.2"
+
+initialData:
+  - collectionName: &collectionName coll
+    databaseName: &databaseName test
+    documents: []
+
+tests:
+  # One-off tests to assert that drivers error if both maxCommitTimeMS and timeoutMS are set.
+
+  - description: "abortTransaction errors if maxCommitTimeMS and timeoutMS are both specified"
+    operations:
+      - name: createEntities
```

Comment:
It doesn't matter for these tests, but the tests that are generated by the `for` loop below need to declare entities on a per-test basis because they configure clients differently. For example, some tests create a client with `socketTimeoutMS` while others create one with `waitQueueTimeoutMS`. I modified the Unified Test Format spec to add a `createEntities` operation to do this (see the changes in `unified-test-format.rst`).